### PR TITLE
add project.janet file to ease the build & install workflow

### PR DESCRIPTION
**Pull request by faywong**

### CODE REVIEW

File: project.janet [hunk 1]
```diff
@@ -0,0 +1,15 @@
+(declare-project
+  :name "janetui"
+  :url "https://github.com/janet-lang/janetui"
+  :repo "https://github.com/janet-lang/janetui.git")
+
+(rule "build/janetui.so" ["CMakeLists.txt"]
+(do
```
Comment (pepe) [inline project.janet:7]: From this line to the end of the form code should be indented. Btw we have formatting stuff in the https://github.com/janet-lang/spork

File: project.janet [hunk 2]
```diff
@@ -0,0 +1,15 @@
+(declare-project
+  :name "janetui"
+  :url "https://github.com/janet-lang/janetui"
```
Comment (pepe) [inline project.janet:3]: Please add a project description.

Comment (faywong) [reply project.janet:3]: with which keyword? :desc?

File: project.janet (hunk 1 above)
Comment (faywong) [reply project.janet:7]: Got it. I will fmt it in the next commit.

File: project.janet (hunk 2 above)
Comment (faywong) [reply project.janet:4]: I add a desc just for document purpose, as jpm doesn't consume it at all.

**Pull request merged**